### ISSUE TITLE
Import md files [Question]

### ISSUE THREAD

**Issue by Ciberbago** [redimp/otterwiki#160]
Hi! I'm testing your software. It seems pretty good, I just have question. Is there a way to import md files so they are recognized? I currently use another app called WikiMD and created some articles with it. Trying to copy them to the repository folder in Otter so they are visible in the web interace, the names appear in the pages, but trying to open them results in a 404 error.

![image](https://github.com/user-attachments/assets/62a581fb-1eb9-4145-a36a-1f10ff5aeada)

Is there a way to make this work? Or should I recreate one by one the md files I already have?

Screenshot of opening imported md file:
![image](https://github.com/user-attachments/assets/0429ffc7-0ebd-4015-8a81-0641c60f1e67)

Screenshot of opening md file created within otter:
![image](https://github.com/user-attachments/assets/f2228d3a-8379-4ff5-a352-47fddc32c50b)

I tried reading the docs but could not find a section for this (maybe I'm dumb, sorry about that)

Thanks in advance!

**Comment by redimp** [comment]
Hey @Ciberbago, my guess is the filename case causes these errors. By default An Otter Wiki assumes that all files are lowercase. 

But there is a fix for this! In `Settings > Content and Editing Preferences` check `[x] Retain page name case`.

<img width="630" alt="image" src="https://github.com/user-attachments/assets/7423ccc4-566f-4d2b-80c5-a796d7b96647">

Re-reading the description I think I have to improve it.

**Comment by Ciberbago** [comment]
You are a wizard. Thank you, that was what I needed. After using the option I can see all the md files I copied. 

After that, I was having this message:

```
Not under version control
This page was loaded from the repository but is not added under git version control. Make a commit on the Edit page to add it.
```

And trying to edit the page, saving it would give me a server error, but after digging in the logs I discovered it was because the files I copied had the owner as root, changing that to www-data as the other files for the app, worked beautifully, just putting this here in case someone else migrating finds the same problem.
